salt/states/ssh_known_hosts.py

```diff
@@ -12,6 +12,7 @@
         - present
         - user: root
         - fingerprint: 16:27:ac:a5:76:28:2d:36:63:1b:56:4d:eb:df:a6:48
+        - fingerprint_hash_type: md5
```

Comment:
The current line 14 here is [the hexed md5 format of GitHub's fingerprint](https://help.github.com/articles/github-s-ssh-key-fingerprints/).

They could be two different examples. There currently isn't a sha256 example in the docs anywhere that I've seen.